base-image/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+
+BUILD_DIR='/home/builder/tmp'
+
+mkdir -p $BUILD_DIR
+cp -r $SOURCE_DIR/* $BUILD_DIR/
+cd $BUILD_DIR
+source poky/oe-init-build-env build
+bitbake -k core-image-minimal
```

Comment:
I had some issues with sanity check from yocto and this flag gets rid of them but we have a clean build now so I will remove it